doc/howto/instances_create.md

```diff
@@ -94,3 +94,29 @@ To launch a container with this instance type, enter the following command:
     lxc launch images:ubuntu/22.04 my-instance --type t2.micro
 
 The list of supported clouds and instance types can be found at [`https://github.com/dustinkirkland/instance-type`](https://github.com/dustinkirkland/instance-type).
+
+### Launch a VM that boots from an ISO
+
+To launch a VM that boots from an ISO, you first need to create a VM. In this scenario it might make sense to create an empty VM as the ISO image could be used for installation. An empty VM can be created using the following command:
+
+    lxc init iso-vm --empty --vm
+
+The second step it to import an ISO image which can later be attached to the VM as a storage volume:
+
+    lxc storage volume import path-to-image.iso iso-volume --type=iso
+
+Lastly, you need to attach the custom ISO volume to the VM using the following command:
+
+    lxc storage volume attach <pool_name> iso-volume iso-vm
+
+Now it's time to start the VM. By default the VM will try booting from its disk. For the VM to boot from the attached ISO, you need to be connected to the console console, and repeatedly press `ESC` until the firmware menu appears:
+
+    lxc start iso-vm --console
+
+Once you have entered the firmware menu, select `Boot Manager`, and `UEFI QEMU QEMU CD-ROM`. If you have multiple ISO images attached to the VM, the entries in the `Boot Manager` will be `UEFI QEMU QEMU CD-ROM`, `UEFI QEMU QEMU CD-ROM 2`, and so on.
+
+The VM will now boot from the ISO image. You will likely need to disconnect from the serial console using `ctrl+a-q`, and connect to the VGA console using the following command:
+
+    lxc console iso-vm --type=vga
+
+You should now see the installer. After the installation in done, the VM will reboot, and boot to disk without you needing to detach the custom ISO volume first.
```

Comment:
```suggestion
You should now see the installer. After the installation is done, the VM will reboot.
It will automatically boot to disk (you don't need to detach the custom ISO volume).
```

Reply:
If we add the disk with boot.priority then we should instruct the user to remove the disk afterwards.